assets/js/blocks/mini-cart/utils/set-styles.ts

```diff
@@ -0,0 +1,53 @@
+function getClosestColor(
+	element: Element | null,
+	colorType: 'color' | 'backgroundColor'
+): string | null {
+	if ( ! element ) {
+		return null;
+	}
```

Comment:
I suggest removing this check. In this way `element` can be defined. The caller should do this check. (you already do for `badgeTextColor` and `badgeBackgroundColor`).